reward/staking_manager.go

```diff
@@ -108,3 +108,7 @@ func (sm *stakingManager) handleChainHeadEvent() {
 		}
 	}
 }
+
```

Comment:
```suggestion

// UnSubscribe can unsubscribe a subscription to listen chain head event..
```

Reply:
Do you have any reason for writing `UnSubscribe` instead of `Unsubscribe`?

Reply:
I've changed it..